components/polyfill_uuid.rst

```diff
@@ -0,0 +1,57 @@
+.. index::
+    single: Polyfill
+    single: PHP
+    single: Components; Polyfill
+
+The Symfony Polyfill / UUID Component
+========================================
+
+    This component provides ``uuid_*`` functions to users who run PHP versions
+    without the UUID extension.
+
+Installation
+------------
+
+.. code-block:: terminal
+
+    $ composer require symfony/polyfill-uuid
+
+.. include:: /components/require_autoload.rst.inc
+
+Usage
+-----
+
+Once this component is installed in your application, you can use the following
+functions, no matter if the `PHP UUID extension`_ is installed or not in your
+server.
+
+Provided Constants
+~~~~~~~~~~~~~~~~~~
+
+* ``UUID_VARIANT_NCS`` (value = 0)
```

Comment:
Maybe a table would fit better here? Maybe same for the other polyfills?

Reply:
I prefer to not do that because tables are hard to maintain in RST docs.